content/tutorial-escrow.md

```diff
@@ -539,6 +549,36 @@ _Websocket_
 
 <!-- MULTICODE_BLOCK_END -->
 
-**Tip:** If you don't know what `OfferSequence` to use in the [EscrowFinish transaction][] to execute an escrow, use the [`tx` method](reference-rippled.html) to look up the transaction that created the escrow, using the identifying hash of the transaction in the Escrow's `PreviousTxnID` field. Use the `Sequence` value of that transaction as the `OfferSequence` value when finishing the escrow.
+###Look up escrows by receiver
+
+You can use the [`account_objects`](reference-rippled.html#account-objects) method to look up escrows by receiver address.
+
+Look up by receiver is possible for only pending escrows created after [fix1523](concept-amendments.html#fix1523) was enabled for rippled 0.80.0 on 2017-11-14.
```

Comment:
Change this from passive to active voice:

> You can only look up pending escrows by receiver if those escrows were created after the [fix1523 amendment](concept-amendments.html#fix1523) was enabled on 2017-11-14.

You may also note in the above suggested revision: it's _slightly_ inaccurate to say that the amendment was enabled for `rippled` 0.80.0. A more precise explanation is that it was enabled for the XRP Ledger. `rippled` is the server that manages the XRP ledger, and technically the amendment applies to the whole ledger regardless of which server implementation is running it and what software version they're on. (`rippled` is the only server implementation for the XRP Ledger, but it's theoretically possible for there to be others, and at one point we even aspired for there to be other implementations.)

Reply:
Done! Thank you for the clarification of rippled vs the XRP Ledger. Fantastic. 🥇